Formula/webdis.rb

```diff
@@ -0,0 +1,37 @@
+class Webdis < Formula
+  desc "Redis HTTP interface with JSON output"
+  homepage "http://webd.is/"
+  url "https://github.com/nicolasff/webdis/archive/0.1.2.tar.gz"
+  sha256 "8e46093af006e35354f6b3d58a70e3825cd0c074893be318f1858eddbe1cda86"
+
+  depends_on "libevent"
+
+  def install
+    system "make"
+    bin.install "webdis"
+
+    inreplace "webdis.prod.json" do |s|
+      s.gsub! "/var/log/webdis.log", "#{var}/log/webdis.log"
```

Comment:
> Do not argue with Homebrew maintainers. You may disagree but unless they change their mind please implement what they request. Ultimately they control what is included in Homebrew as they have to support any changes that are made.

https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#following-up

> Documentation tends to lag behind current decision-making. Although some rejections may seem arbitrary or strange they are based from years of experience making Homebrew work acceptably for our users.

https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#sometimes-there-are-exceptions

The reason we have text like these is because sometimes we may need to ask for things in review that are not already documented or checked. I had made a TODO to update both the documentation and the check for this, believe it or not. If you're going to use quote marks please quote what I actually said: "We can argue about this all you like but it'd go quicker if you could just change it, thanks.". I said it _would go quicker_ which it would have. If I'd have known I'd be called names as a result I wouldn't have bothered reviewing this PR at all.